### DISCUSSION TITLE
Please change Wraith King's arcana gem requirements.

### DISCUSSION THREAD

**Discussion by DedSilent** [ValveSoftware/Dota2-Gameplay#16074] [General]
It is by far the most frustrating arcana to farm. I've gotten all the way up to 90 heroes current;ly, and as I expected, the progress has slowed down to a point where I just want to abandon the self-sought mission altogether.

NINE games. Nine games I played once I got home, up into the morning, trying to get even just ONE hero to progress my gem. Lost 6 of them, and 2 of the 3 that I won actually had heroes I needed, but I was unable to get the final blow on them.

You literally have to pray that the stars align before you even get a CHANCE to potentially make any progress. It's far from tedious at this point. Tedious is more defining of the gems that you CAN make progress on in a reliably consistent fashion. But this gem is just completely egregious. 

I'll have games where I do land the killing blow, but for one reason or another we end up losing the match. And then I go another two or three games, win or lose, before I get the chance to make progress again. Only to fail again. And then your mind stirs with how long you'll have to wait before yet another chance arrives.

That was my experience tonight. And that's JUST tonight.

Why must the gems be such headaches to farm? Why can't it just be simple and consistent like Earthshaker's or Ogre's? 

It's so excruciatingly disheartening, to sit down for an entire play session, prolonged, and fail to make even an inkling of progress. Day after day, and match after match. I fail to complete one of the MANY requirements needed to make any progress whatsoever on this gem. 

**Can this gem please be changed?**

**Comment by commentator24** [comment]
ohhh i remember the pain. initially I keep a list of heroes i need to do, but this didn't work well. i ended up creating a list of heroes i unlocked and still need to kill to clearly see my options.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/8f6fd376-0172-4a8b-9946-212b0f89356d)
to get the final blow I made dagon 5 and just ks. after that it's very easy.

I cant remember if turbo works, but i think it works? a lot easier in turbo.
I didn't do these: but one way is to avoid one of your friend (if you have dotaplus) then queue at the same time e.g. turbo / captain's mode? until you found match together. Ask your friend (or use another laptop?) to play the hero you need and purposely lose :)

the final way is to gather 10 ppl in a lobby, shuffle to balance and create 2 party to queue same time e.g. captain's mode. ask enemies to play 5 heroes you need to win with.
